packages/storybook/src/stories/Sidebar/Sidebar.stories.tsx

```diff
@@ -0,0 +1,171 @@
+import type { Meta } from '@storybook/react';
+import { withRouter } from 'storybook-addon-react-router-v6';
+
+import {
+	FileTextIcon,
+	FileCheckmarkIcon,
+	FolderMinusIcon,
+	TrashIcon,
+	CogIcon,
+	MagnifyingGlassIcon,
+	PlusIcon,
+	InboxFillIcon,
+} from '@navikt/aksel-icons';
+
+import { SidebarItem } from '../../../../frontend-design-poc/src/components/SidebarItem';
+import { Sidebar } from '../../../../frontend-design-poc/src/components/Sidebar';
+import { HorizontalLine } from '../../../../frontend-design-poc/src/components/Sidebar/Sidebar';
```

Comment:
Now that the PR on updated import statements are merged we might update the import statements in this PR.

Reply:
Good point,  I missed that.